src/client/src/apps/MainRoute/routes/BlotterRoute.tsx

```diff
@@ -9,10 +13,21 @@ const BlotterContainerStyle = styled('div')`
   margin: auto;
 `
 
-const BlotterRoute = () => (
-  <BlotterContainerStyle>
-    <BlotterContainer />
-  </BlotterContainerStyle>
-)
+const getFilterFromQueryStr: (queryStr: string) => BlotterFilter =
+  queryStr => {
+    const parsedQueryString = queryString.parse(queryStr)
```

Comment:
now opening URL like this `http://localhost:3000/blotter?dealtCurrency=USD,dealtCurrency=GBP` will filter blotter by two currencies `USD` and `GBP` (logical OR)